src/tests/handlers/websocket/message_test.py

```diff
@@ -0,0 +1,38 @@
+from parameterized import parameterized
+from src.tests.data.data_reader import read
+from unittest import TestCase, mock
+from src.handlers.websocket.message import handler, get_api_url
+
+
+class TestUploadDataS3TriggerHandler(TestCase):
```

Comment:
I think this class have same name as the class in 'src/tests/handlers/upload_file/upload_file_s3_handler_test.py'